tapes/bleu.tape

```diff
@@ -33,12 +33,13 @@ task multi_bleu : mosesdecoder
   fi
 }
 
-task sacrebleu
+task sacrebleu : sacrebleu
     < in=$detokenized_output
     < ref=$out@dummy_aggregate_merge[DataSection:devtest,DevtestDataSection:test,side:trg]
```

Comment:
I would suggest that instead of removing the XML-related tasks we change the input source here from `dummy_aggregate_merge` to `donwload_or_link`. Or if sacrebleu is already configured to download the test set, we may as well just pass a configuration variable indicating which test set we are intending to use.

Reply:
That makes sense.